src/server/game/Entities/Player/Player.cpp

```diff
@@ -1717,6 +1717,7 @@ bool Player::TeleportTo(uint32 mapid, float x, float y, float z, float orientati
                 UnsummonPetTemporaryIfAny();
         }
 
+
```

Comment:
Redundant blank line, please remove
(unless there is more code to be added later).

Reply:
Done.